extensions/mssql/src/main.ts

```diff
@@ -203,24 +203,21 @@ async function handleOpenNotebookTask(profile: azdata.IConnectionProfile): Promi
 	}
 }
 
-async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionProfile, appContext: AppContext): Promise<void> {
-	const notebookRelativePath: string = 'notebooks/tsg/cluster-status.ipynb';
-	const notebookFullPath: string = path.join(appContext.extensionContext.extensionPath, notebookRelativePath);
-	if (!(await Utils.exists(notebookFullPath))) {
-		vscode.window.showErrorMessage(localize("fileNotFound", "Unable to find the file specified"));
-	} else {
-		const title: string = Utils.findNextUntitledEditorName(notebookFullPath);
-		const untitledFileName: vscode.Uri = vscode.Uri.parse(`untitled:${title}`);
-		vscode.workspace.openTextDocument(notebookFullPath).then((document) => {
-			let initialContent = document.getText();
-			azdata.nb.showNotebookDocument(untitledFileName, {
-				connectionProfile: profile,
-				preview: true,
-				initialContent: initialContent,
-				initialDirtyState: false
-			});
-		});
+async function handleOpenClusterDashboardTask(profile: azdata.IConnectionProfile, appContext: AppContext): Promise<void> {
+	const serverInfo = await azdata.connection.getServerInfo(profile.id);
+	const controller = Utils.getClusterEndpoints(serverInfo).find(e => e.serviceName === Endpoint.controller);
+	if (!controller) {
+		appContext.apiWrapper.showErrorMessage(localize('noController', "Could not find the Controller endpoint for this instance"));
+		return;
 	}
+
+	appContext.apiWrapper.executeCommand('bigDataClusters.command.manageController',
+		{
+			url: controller.endpoint,
+			auth: profile.authenticationType === 'Integrated' ? AuthType.Integrated : AuthType.Basic,
```

Comment:
i'm a little concerned that we have 'Integrated' in some places and 'integrated' in others. may want to do a toLowerCase() here just to be safe.

Example (extensions/big-data-cluster/src/extension.ts):

		auth: profile.authenticationType === 'SqlLogin' ? 'basic' : 'integrated',

Reply:
This is a general problem we have - for some reason the authentication type is just a string and not an enum or something so the profile auth type is Integrated while the BDC one is integrated. 

Given that we already have a ton of places in the code where we assume profile authenticationType is Integrated I'm not going to change this one since I don't really think that's a good long term solution anyways and if this ever isn't Integrated we're going to have other problems regardless.